client/src/Utils/StatusUtils/useStatusUtils.ts

```diff
@@ -14,6 +15,8 @@ const useStatusUtils = () => {
     const investigatedPatient = useSelector<StoreStateType, InvestigatedPatient>(state => state.investigation.investigatedPatient);
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
     const userId = useSelector<StoreStateType, string>(state => state.user.id);
+    const investigationEndTime = useSelector<StoreStateType, Date | null>(state => state.investigation.endTime);
+    const wasInvestigationReopend = investigationEndTime !== null;
```

Comment:
maybe use useMemo here

Reply:
That changes only once so I dont think that necessary